dtmi/advantech/icam_500-1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "@id": "dtmi:Advantech:ICAM_500;1",
+  "@type": "Interface",
+  "displayName": "ICAM-500",
+  "contents": [
+    {
+      "@type": "Component",
+      "name": "deviceinfo",
+      "schema": "dtmi:Advantech:JetsonDeviceInformation;1"
+    },
+    {
+      "@type": "Component",
+      "name": "thermalzones",
```

Comment:
Should the name of this component be `thermalZones` or `Thermal_Zones`?

Reply:
updated as suggestion